### PR TITLE
chore: add extra step to deploy switchboard gp4btc clone

### PR DESCRIPTION
**Pull request by Timtech4u**
Extra step to deploy switchboard gp4btc clone to switchboard.gp4btc.org

### CODE REVIEW

File: .github/workflows/cf-deploy.yml [hunk 1]
```diff
@@ -115,3 +115,24 @@ jobs:
           -H 'Accept: application/vnd.github.everest-preview+json' \
           -u ${{ secrets.ACCESS_TOKEN }} \
           --data '{"event_type": "ping", "client_payload": { "repository": "'"$GITHUB_REPOSITORY"'" }}'
+      # Deployment to extra Cloudfront/S3 for gp4btc.org domain
```
Comment (jrhender) [inline .github/workflows/cf-deploy.yml]:
```suggestion
      # Deployment to Cloudfront/S3 for gp4btc.org domain 
```

File: .github/workflows/cf-deploy.yml [hunk 2]
```diff
@@ -115,3 +115,24 @@ jobs:
           -H 'Accept: application/vnd.github.everest-preview+json' \
           -u ${{ secrets.ACCESS_TOKEN }} \
           --data '{"event_type": "ping", "client_payload": { "repository": "'"$GITHUB_REPOSITORY"'" }}'
+      # Deployment to extra Cloudfront/S3 for gp4btc.org domain 
+      - name: Configure AWS credentials for GP4BTC Switchboard CLONE
```
Comment (jrhender) [inline .github/workflows/cf-deploy.yml]:
```suggestion
      - name: Configure AWS credentials for gp4btc.org Switchboard
```

File: .github/workflows/cf-deploy.yml [hunk 3]
```diff
@@ -115,3 +115,24 @@ jobs:
...
+        uses: aws-actions/configure-aws-credentials@v1
+        if: ${{ steps.env_vars.outputs.env_name == 'gp4btc-prod' }}
+        with:
+          aws-access-key-id: ${{ secrets.GP4BTC_SWITCHBOARD_CLONE_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.GP4BTC_SWITCHBOARD_CLONE_ACCESS_KEY }}
```
Comment (jrhender) [inline .github/workflows/cf-deploy.yml]:
```suggestion
          aws-access-key-id: ${{ secrets.GP4BTC_DOMAIN_SWITCHBOARD_ACCESS_KEY_ID }}
          aws-secret-access-key: ${{ secrets.GP4BTC_DOMAIN_SWITCHBOARD_CLONE_ACCESS_KEY }}
```

**Review by jrhender — COMMENTED** [review]
I'm okay with the second deployment approach but I think the name of the variables and steps should be such that it would make sense if we remove the "cloned" deployment. I provided some example suggested changes to this effect.

@Timtech4u could you also explain why this is a "hotfix" PR please?

**Comment by jrhender** [comment]
@Timtech4u could you also please check the code style for the PR? https://github.com/energywebfoundation/switchboard-dapp/actions/runs/4925988802/jobs/8800952648?pr=883

**Comment by Timtech4u** [comment]
Thanks @jrhender , I have updated the secret names and it appears the lint passes now. The idea of creating this as a hotfix was because it was targeting production, we can definitely sync it with the staging branch.

**Comment by jrhender** [comment]
Thanks for updating the names @Timtech4u . Could you double check the prettier results please? I think there may still be code style issues: https://github.com/energywebfoundation/switchboard-dapp/actions/runs/4933908842/jobs/8818464943?pr=883

**Comment by Timtech4u** [comment]
Deployment not actively in use imo

**Pull request closed without merging**